lib/gamnit/programs.nit

```diff
@@ -0,0 +1,572 @@
+# This file is part of NIT (http://www.nitlanguage.org).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	 http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Services for graphical programs with shaders, attributes and uniforms
+module programs
```

Comment:
I see it more as `gamnit::programs` and it goes with the main class name `GamnitProgram`. But I'll keep it in mind, I may rename the class to `GraphicProgram` or something like that.